llvm/lib/Analysis/StackSafetyAnalysis.cpp

```diff
@@ -489,7 +489,8 @@ void StackSafetyLocalAnalysis::analyzeAllUses(Value *Ptr,
           US.addRange(I, UnknownRange, /*IsSafe=*/false);
           break;
         }
-        if (const MemIntrinsic *MI = dyn_cast<MemIntrinsic>(I)) {
+        if (const MemIntrinsic *MI = dyn_cast<MemIntrinsic>(I);
```

Comment:
Why stack safety and sanitizers are opt ed out?

Reply:
Matching prior behavior for code I was unfamiliar with.

Reply:
I guess we should either:
1. implement MemsetPatternInst for all cases, before this patch, and just combine handlers in this patch
2. add clear TODO/FIXME, and do 1. later